docs/anforderung/files/iteration-plans/iteration_14_plan.adoc

```diff
@@ -0,0 +1,105 @@
+:sectnums:
+:icons: font
+
+== Meilensteine
+//Meilensteine zeigen den Ablauf der Iteration, wie z.B. den Beginn und das Ende, Zwischen-Meilensteine, Synchronisation mit anderen Teams, Demos usw.
+
+[%header, cols="3,1"]
+|===
+|Meilenstein
+|Datum
+|Beginn der Iteration |03.08.2020
+|Fertigstellen der Aufgaben|-
+|Ende der Iteration	|14.08.2020
+|===
+
+
+== Wesentliche Ziele
+//Nennen Sie 1-5 wesentliche Ziele für die Iteration.
+
+* Essence Navigator weiterhin auf dem Laufenden halten
+* Abgabe vollenden
+
+
+== Aufgabenzuordnung
+//Dieser Abschnitt sollte einen Verweis auf die Work Items List enthalten, die die für diese Iteration vorgesehenen Aufgaben dokumentiert sowie die Zuordnung dieser Aufgaben zu Teammitgliedern. Alternativ können die Aufgaben für die Iteration und die Zuordnung zu Teammitgliedern in nachfolgender Tabelle dokumentiert werden - je nach dem, was einfacher für die Projektbeteiligten einfacher zu finden ist.
+
+Die in dieser Iteration geplanten Aufgaben sind in der Work Items List dargestellt (https://github.com/mribrgr/StuRa-Mitgliederdatenbank/issues).
+
+_alternativ:_ Die folgenden Aufgaben werden in dieser Iteration bearbeitet:
+[%header, cols="3,1,1,1,2,1,1,1"]
+|===
+|Aufgabe bzw. Beschreibung	|Priorität  	|Schätzung der Größe (Punkte) | Status | Referenzen | Zugewiesen (Name) |	Gearbeite Stunden | Schätzung der verbleibenden Stunden
+
+// wie immer
+| Essence Navigator | niedrig | 2 | erledigt | link:../../img/essenceNavigator[Essence Navigator Bilder], link:https://github.com/mribrgr/StuRa-Mitgliederdatenbank/issues/25[Issue #25], link:https://github.com/mribrgr/StuRa-Mitgliederdatenbank/issues/26[Issue #26] | mribrgr | 2 | 0
+
+// alte Aufgaben
+
+// neue Aufgaben
+// thrs79136, yoshinonxkun, mribrgr, benjamin-hempel, redifity
+| Aufgabenstellung, Auftraggeber, Ausgangssituation | sehr hoch | 3 | erledigt | x | mribrgr | 5 | 0
+| Eingesetzte Techniken und Praktiken | sehr hoch | 5 | erledigt | x | redifity | 8 | 0
+| Ziele, Aktivitäten, Ergebnisse der Iterationen | sehr hoch | 5 | erledigt | x | mribrgr | 8 | 0
+| Anforderungserhebung und -analyse | sehr hoch | 5 | erledigt | x | thrs79136 | 8 | 0
+| Entwurf | sehr hoch | 5 | erledigt | x | redifity | 8 | 0
+| Test | sehr hoch | 5 | erledigt | x | yoshinonxkun | 8 | 0
+| Übergabe | sehr hoch | 5 | erledigt | x | yoshinonxkun | 8 | 0
+| Dokumentation | sehr hoch | 5 | erledigt | x | yoshinonxkun | 8 | 0
+| Probleme und Umgang mit diesen | sehr hoch | 5 | erledigt | x | thrs79136 | 8 | 0
+| Projektergebnisse | sehr hoch | 5 | erledigt | x | thrs79136 | 8 | 0
+| Projektorganisation | sehr hoch | 5 | erledigt | x | benjamin-hempel | 8 | 0
+| Implementierung | sehr hoch | 5 | erledigt | x | benjamin-hempel | 8 | 0
+| Wesentliche Entscheidungen | sehr hoch | 5 | erledigt | x | benjamin-hempel | 8 | 0
+| Iterationspläne erneut überprüfen | sehr hoch | 3 | erledigt | x | redifity | 5 | 0
+| Reflexion pro Teammitglied | sehr hoch | 5 | erledigt | x | alle | 8 | 0
```

Comment:
Finde hier die Stundenanzahlen ziemlich hoch angesetzt. Ich für meinen Teil hab in der letzten Iteration definitiv nicht 24h für meinen Teil des Projektberichts aufgewendet :D

Reply:
ich auch nicht

Reply:
Es sind auch generell viele Aufgaben in dieser Iteration

Reply:
@benjamin-hempel wie kommst du auf 24h?

Reply:
> Es sind auch generell viele Aufgaben in dieser Iteration

Gibt es wenig dran zu ändern, war eben so 🙈

Reply:
@mribrgr In der Spalte "Gearbeitete Stunden" stehen für meinen Teil des Projekts 3x 8h = 24h

Reply:
behoben,lol